code/mob/living/silicon/ai.dm

```diff
@@ -383,6 +384,10 @@ or don't if it uses a custom topopen overlay
 			src.bioHolder.mobAppearance.pronouns = src.client.preferences.AH.pronouns
 			src.update_name_tag()
 
+		src.camera = new /obj/machinery/camera(src)
+		src.camera.c_tag = src.real_name
```

Comment:
Minor thing but this probably should update on name change then?

Reply:
Afaik theres no actual way to see this camera in a list naturally, and borg name changes dont update it I think

Reply:
Can confirm borg name changes dont change their camera name and are probably "Cyborg Beta-15" forever but i'm not entirely sure it matters as there is no way to see these cameras on a list unlike "Test Chamber 1" for example